web/impact/impact/v1/metadata.py

```diff
@@ -59,6 +59,24 @@
     }
 }
 
+ORGANIZATION_HISTORY_ACTIONS = {
+    "GET": {
+        "history": {
+            "type": "field",
+            "read_only": True,
+        }
+    }
+}
+
+USER_HISTORY_ACTIONS = {
```

Comment:
if these are going to output the same I would just define a single 'HISTORY_ACTIONS' constant and avoid the duplication until necessary.